nifi-nar-bundles/nifi-lumberjack-bundle/nifi-lumberjack-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,28 @@
+nifi-lumberjack-nar
+Copyright 2014-2015 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) 
+    The following NOTICE information applies:
+      The Netty Project
+      Copyright 2014 The Netty Project
+
+******************
+The MIT License (MIT)
+******************
+
+The following source code components are provided under the MIT License (MIT)
+
+  (MIT) 
+    The following NOTICE information applies:
+      Graylog Logstash forwarder Lumberjack Input Plugin
```

Comment:
Wasn't sure what this was referencing.  Did not see any explicit dependencies incorporated, was there source that was copied into the project?